### DISCUSSION TITLE
Using the Pybricks library in standard python, how do i intitialize an Inventor Hub

### DISCUSSION THREAD

**Discussion by ihavenoideawhatimdoingh** [pybricks/support#1607] [Q&A]
so i have this simple script

```
from pybricks.hubs import InventorHub
from pybricks.pupdevices import Motor, ColorSensor, UltrasonicSensor
from pybricks.parameters import Button, Color, Direction, Port, Side, Stop
from pybricks.robotics import DriveBase
from pybricks.tools import wait, StopWatch
import pyautogui

hub = InventorHub()
while True:
    file1 = open("myfile.txt","w+")
    L = []
    file1.writelines(L)
    file1.close()
    file1 = open("myfile.txt","r")
    print(hub.buttons.pressed())
    file1.close()

```
i need to use standard python to use pyautogui so if someone has any way of helping me initialize the hub, please tell me (ive already connected via bluetooth)

**Comment by ihavenoideawhatimdoingh** [comment]
btw ik that the writen lines are nothing but im talking about the print

**Comment by dlech** [comment]
Pybricks run on MicroPython on the hub, it doesn't run using standard Python or on your computer.

You can make a program that communicates from the hub to your computer though.

**Reply by ihavenoideawhatimdoingh** [reply]
https://pypi.org/project/pybricks/#description ? and if this is like non functional anymore or whatever, how would i do that?

**Reply by laurensvalk** [reply]
Here is an example of combining Python code on your PC with Pybricks MicroPython running on your hub: https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication/

**Reply by dlech** [reply]
The `pybricks` package on PyPI is just for code completion. It does nothing at runtime.
